libfoundation/include/foundation/sys/dbglog.h

```diff
@@ -71,7 +72,7 @@ static inline const char* _isix_dbglog_extract_basename(const char *file) {
 		 usart_init(__VA_ARGS__); } while(0)
 
 #else /* PDEBUG */
-
+#define dbonly(code)
```

Comment:
What is this macro and what is it supposed to do?

Reply:
It's a helper mainly to avoid unused variables warnings in non-debug without #ifdefs, e.g.
```c++
dbprintf("Read test block size %u", bs );
dbonly(ostick_t begin = isix::get_jiffies();)
for(size_t c=0; c<N_sects; c+=(bs/512) )
{
ret = card->read( buf, c, bs/512 );
if( ret )
{
	dbprintf("Read error with code %i", ret);
	return;
}
}
dbonly(ostick_t time = isix::get_jiffies() - begin;)
dbprintf("Speed %u kb/s", (1000*(N_sects/2)) / time);
```
It came up in quite a few places in isixsamples.

Reply:
ah ok.